src/prompt.js

```diff
@@ -3,9 +3,25 @@ const path = require('path');
 
 const promptFile = path.join(__dirname, 'prompt.txt');
 
+const fractalTestFile = path.join(
+  __dirname,
+  '../transcripts/noc-8-fractals.md'
+);
+const fractalTest = fs.readFileSync(fractalTestFile, `utf8`);
+
 const prompts = {
   startingPrompt: `Write the code for a fractal tree in p5.js`,
   systemPrompt: fs.readFileSync(promptFile, `utf8`),
+  promptingTemplate:
+    'Dan says: {prompt}\nCurrent code in the editor:\n```\n{currentCode}\n```',
+  promptingWithContextTemplate:
+    'Here is Chapter 8 from the Nature of Code that you can use as a reference for your fractal tutorial: /' +
+    fractalTest +
+    +'\n\n' +
+    fs.readFileSync(promptFile, `utf8`) +
```

Comment:
(this will add the entire system prompt in every message, is that intended?)

Reply:
Yes, but it's not my intention to keep it this way, I'm just messing around and found that if I include the entire Nature of Code chapter (super long "context") after the system prompt the agent stopped following the other instructions in terms of how to write the code. I was thinking that given the larger context windows for GPT4o and Gemini 1.5 that I could pass along full transcripts of  videos or full chapters / tutorials for information for the agent to draw on.